docs/resources/grants.md

```diff
@@ -138,7 +138,7 @@ resource "databricks_grants" "things" {
 
 ## View grants
 
-You can grant `ALL_PRIVILEGES`, `APPLY_TAG` and `SELECT` privileges to [_`catalog.schema.view`_](views.md) specified in `table` attribute.
+You can grant `ALL_PRIVILEGES`, `APPLY_TAG` and `SELECT` privileges to [_`catalog.schema.view`_](../data-sources/views.md) specified in `table` attribute.
```

Comment:
same here

Reply:
Should `catalog.schema.view` point to `sql_table.md` as well as `catalog.schema.table`? can't find `*view.md` in `resources` directory.

Reply:
yes, view is created via `sql_table`